src/components/SideBar.tsx

```diff
@@ -29,7 +33,14 @@ const CourseItem = styled.li`
 		color: ${({ theme }) => theme.colors.text[1]};
 		list-style: none;
 	}
+	${(props) =>
+		props.selected &&
+		css`
+			background-color: ${({ theme }) => theme.colors.secondary[1]};
+			color: ${({ theme }) => theme.colors.text[1]};
```

Comment:
Looks great! I am wondering if we should add some spacing between hover colour and selection colour.

Specifically, on the CourseItem there is a `padding-bottom: 1em; padding-top: 1em` and I wonder if we should change the padding to something like `0.75em` and add a `0.25em` margin on the top and bottom (so when hovering next to the selected item, the highlighted background won't touch)

Reply:
Tried out the suggestion and it does look much better with the margin! I pushed the change.